### PR TITLE
refactor: OpenRewrite best practices

### PR DESCRIPTION
**Pull request by timtebeek**
Use this link to re-run the recipe: https://app.moderne.io/recipes/org.openrewrite.recipes.OpenRewriteBestPractices?organizationId=T3BlblJld3JpdGU%3D

### CODE REVIEW

**Comment by timtebeek** [comment]
@mike-solomon didn't get to finishing this earlier; would you be able to help add example values for the 21 TODO comments inserted here?

File: rewrite-core/src/main/java/org/openrewrite/FindParseFailures.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public class FindParseFailures extends Recipe {
     @Nullable
     String parserType;
 
-    @Option(displayName = "Stack trace",
+    @Option(example = "TODO Provide a usage example for the docs", displayName = "Stack trace",
```
Comment (timtebeek) [inline rewrite-core/src/main/java/org/openrewrite/FindParseFailures.java:44]:
```suggestion
    @Option(example = "RuntimeException", displayName = "Stack trace",
```

File: rewrite-gradle/src/main/java/org/openrewrite/gradle/AddProperty.java [hunk 2]
```diff
@@ -39,7 +39,7 @@ public class AddProperty extends ScanningRecipe<AddProperty.NeedsProperty> {
             example = "org.gradle.caching")
     String key;
 
-    @Option(displayName = "Property value",
+    @Option(example = "TODO Provide a usage example for the docs", displayName = "Property value",
```
Comment (timtebeek) [inline rewrite-gradle/src/main/java/org/openrewrite/gradle/AddProperty.java:42]:
```suggestion
    @Option(example = "true", displayName = "Property value",
```

File: rewrite-gradle/src/main/java/org/openrewrite/gradle/UpdateGradleWrapper.java [hunk 3]
```diff
@@ -94,13 +94,13 @@ public String getDescription() {
...
 
     @Getter
-    @Option(displayName = "Wrapper URI",
-            description = "The URI of the Gradle wrapper distribution. " +
-                          "Lookup of available versions still requires access to https://services.gradle.org " +
-                          "When this is specified the exact literal values supplied for `version` and `distribution` " +
-                          "will be interpolated into this string wherever `${version}` and `${distribution}` appear respectively. " +
-                          "Defaults to https://services.gradle.org/distributions/gradle-${version}-${distribution}.zip.",
-            required = false)
+            @Option(example = "TODO Provide a usage example for the docs", displayName = "Wrapper URI",
```
Comment (timtebeek) [inline rewrite-gradle/src/main/java/org/openrewrite/gradle/UpdateGradleWrapper.java:97]:
```suggestion
            @Option(example = "https://services.gradle.org/distributions/gradle-${version}-${distribution}.zip", displayName = "Wrapper URI",
```

**Comment by mike-solomon** [comment]
@timtebeek Yep - I believe I fixed all open TODOs. Please let me know if you run into any issues with these. I'm not actually sure how this will appear but I did my best to put in examples as I found them in tests or online.

**Review by timtebeek — COMMENTED** [review]
Thanks a lot for helping fill out the examples Mike!

**Pull request merged**